pootle/apps/pootle_store/store/serialize.py

```diff
@@ -0,0 +1,75 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) Pootle contributors.
+#
+# This file is a part of the Pootle project. It is distributed under the GPL3
+# or later license. See the LICENSE file for a copy of the license and the
+# AUTHORS file for copyright and authorship information.
+
+from django.core.cache import caches
+from django.utils.functional import cached_property
+
+from pootle.core.delegate import config, serializers
+
+
+class StoreSerializer(object):
+
+    def __init__(self, store):
+        self.store = store
+
+    @property
+    def project_serializers(self):
+        project = self.store.translation_project.project
+        return (
+            config.get(
+                project.__class__,
+                instance=project,
+                key="pootle.core.serializers")
+            or [])
+
+    @property
+    def pootle_path(self):
+        return self.store.pootle_path
+
+    @cached_property
+    def max_unit_revision(self):
```

Comment:
hmm, maybe but tbh i dont like adding things like that on models - imho this most likely needs to be denormalized